src/view/DocumentView.cpp

```diff
@@ -116,7 +125,14 @@ void DocumentView::drawTexImage(cairo_t* cr, TexImage* texImage) {
 
         cairo_translate(cr, texImage->getX(), texImage->getY());
         cairo_scale(cr, xFactor, yFactor);
-        poppler_page_render(page, cr);
+        if(this->markAudioStroke){ // make non-Audio strokes translucent when markAudioStroke
+            cairo_push_group(cr); // switch to temporary surface
+            poppler_page_render(page, cr); // render page
+            cairo_pop_group_to_source(cr); // return to original surface and set temp surface as pattern
+            cairo_paint_with_alpha(cr, AudioElement::opacityNoAudio / 255.0); // paint the temporary surface w/ opacity level
```

Comment:
Thank you for adding comments! I think this would be more readable if the comments were on a separate line from the code.
```suggestion
        // make non-Audio strokes translucent when markAudioStroke
        if(this->markAudioStroke){
            // Switch to a temporary surface, render the page, then switch back. This sets the
            // temporary surface to the current pattern.
            cairo_push_group(cr);
            poppler_page_render(page, cr);
            cairo_pop_group_to_source(cr);
            
            // paint the temporary surface w/ opacity level
            cairo_paint_with_alpha(cr, AudioElement::opacityNoAudio / 255.0);
```